language-extensions/java/src/linux/JavaExtensionUtils_linux.cpp

```diff
@@ -163,6 +163,25 @@ JavaExtensionUtils::fn_createJvm JavaExtensionUtils::LoadJvm(const string& jvmPa
 	return reinterpret_cast<fn_createJvm>(dlsym(g_jvmLib, "JNI_CreateJavaVM"));
 }
 
+//----------------------------------------------------------------------------
+// Name: JavaExtensionUtils::ShutdownJvm
+//
+// Description:
+//  Cleans up JVM resources. Doesn't call Invocation API function
```

Comment:
nit: if there was another iteration, we may add In Linux, it doesn't call ... (to remind we do this in windows)

Reply:
updated.